load_1D.py

```diff
@@ -0,0 +1,108 @@
+import re
+import numpy as np
+
+
+def load_1D(filepath):
+    """
+    Loads *.ccl or *.dat file (Distinguishes them based on last 3 chars in string of filepath
+    to add more variables to read, extend the elif list
+    the file must include '#data' and number of points in right place to work properly
+
+    :arg filepath
+    :returns det_variables
+    - dictionary of all detector/scan variables and dictinionary for every measurement.
+    Names of these dictionaries are M + measurement number. They include HKL indeces, angles, monitors,
+    stepsize and array of counts
+    """
+    try:

```

Comment:
I would omit `try, except` here, because it doesn't improve the error message. Compare, with the situation without this `try, except`:
```
FileNotFoundError: [Errno 2] No such file or directory: 'test'
```

Reply:
Alright, try, except deleted. I put those there to go around exceptions which I'm not yet sure how to solve so it doesent crash the execution of modules.